src/routes/App.tsx

```diff
@@ -1,23 +1,97 @@
-import logo from '../logo.svg';
-import './App.css';
+import React from "react";
+// import logo from "./logo.svg";
+import "./App.css";
+import Stat from "../components/Stat";
+import Card from "../components/Card";
 
 function App() {
+  // TODO: Don't know a better way to do this yet
+  document.documentElement.style.height = "fill-available";
+  document.documentElement.classList.add("bg-color-primary-background");
+  // document.body.style.height = "100vh";
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p className='weight'>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App flex flex-col h-screen bg-color-primary-background font-Roboto content-center md:justify-center md:gap-20">
```

Comment:
Nit: does this need to be a flex box? Feels over complicated

Reply:
Removed and added top margin to card container